appveyor/appveyor.ps1

```diff
@@ -50,7 +50,7 @@ if(!(Test-Path $simulacron_jar)) {
 }
 
 # Install Python Dependencies for CCM.
-Start-Process python -ArgumentList "-m pip install psutil pyYaml six numpy" -Wait -NoNewWindow
+Start-Process python -ArgumentList "-m pip install psutil pyYaml numpy" -Wait -NoNewWindow
```

Comment:
This one probably should stay.  The usage here is to add the dependencies that ccm needs (in order to support starting and stopping clusters for unit tests) and as of this writing it [does still require six](https://github.com/riptano/ccm/blob/master/requirements.txt).  It's not a _huge_ thing since we don't really rely on AppVeyor much at the moment but I also don't want to delete it as it might cause some level of confusion later.